elasticgraph-warehouse/sig/elastic_graph/warehouse/schema_definition/factory_extension.rbs

```diff
@@ -2,6 +2,7 @@ module ElasticGraph
   module Warehouse
     module SchemaDefinition
       module FactoryExtension: ::ElasticGraph::SchemaDefinition::Factory
+        def new_results: () -> ::ElasticGraph::SchemaDefinition::Results
```

Comment:
```suggestion
```

This shouldn't be needed since it's an override and the method signature already exists on `::ElasticGraph::SchemaDefinition::Factory`.